Authentication/LogIn/View/LoginController.swift

```diff
@@ -48,10 +55,12 @@ public final class LoginController: UIViewController {
     init(viewModel: LoginViewModelType,
         loginViewFactory: () -> LoginViewType,
         onRegister: (LoginController) -> (),
-        delegate: LoginControllerDelegateType = DefaultLoginControllerDelegate()) {
+        onRecoverPassword: (LoginController) -> (),
```

Comment:
The thing is if the user instead of presenting the register controller wants to directly change the rootviewcontroller of the window. The delegate doesn't have the window. Maybe we could have a mutable property that changes when register is selected and set the authentication bootstrapper to listen to it, and use the delegate for injecting more behaviour. Nevertheless, i believe there should be a default everything at all steps, even if it is an "empty" delegate.